broken-plugin.mjs

```diff
@@ -0,0 +1,53 @@
+import { list, sideEffect, context, lambda } from "postgraphile/grafast";
+import { withPgClient } from "postgraphile/@dataplan/pg";
+import {
+  gql,
+  makeExtendSchemaPlugin,
+  makeWrapPlansPlugin,
+} from "postgraphile/utils";
+
+export const BrokenWrap = makeWrapPlansPlugin(
+  (context) => {
+    if (context.scope.isRootQuery || context.scope.isRootMutation) {
+      return { scope: context.scope };
+    }
+    return null;
+  },
+  ({ scope }) =>
+    (plan) => {
+      const $userId = context().get("userId");
+      sideEffect($userId, () => {});
```

Comment:
If this does `return null` then we get `null` instead of `undefined` in the extra value in the `hoofer` array. (The one that shouldn't be there.)